src/features/author.ts

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable no-param-reassign */
```

Comment:
why are there 2 variants of slices - in store/reducers and here - in features?
Delete one